src/components/Book.js

```diff
@@ -0,0 +1,25 @@
+import { useDispatch } from 'react-redux';
+import { removeBooks } from '../redux/books/books';
+
+const NewBook = (props) => {
```

Comment:
- [ ] It would be better if the add book action works as expected in the UI, currently, when I fill in the inputs with the data or information and hits the submit button, the data is not submitted and the UI does not update. Because of this, I am unable to test the remove book action